diff --git a/pyproject.toml b/pyproject.toml
index 0c40083208..4d9616b6f9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -9,7 +9,10 @@ packages = [
     { include = "litellm" },
     { include = "litellm/py.typed"},
 ]
-
+exclude = [
+    "litellm/proxy/enterprise",
+    "litellm/proxy/enterprise/*"
+]
 [tool.poetry.urls]
 homepage = "https://litellm.ai"
 repository = "https://github.com/BerriAI/litellm"
